core/modules/replica/testConfiguration.cc

```diff
@@ -97,6 +104,10 @@ BOOST_AUTO_TEST_CASE(ConfigurationTest) {
         {"worker:worker-B.fs_host",      "host-B"},
      // {"worker:worker-B.fs_port",      "52002"},      // assuming default
         {"worker:worker-B.data_dir",     "/data/B"},
+        {"worker:worker-B.db_host",      "host-B"},
+     // {"worker:worker-B.db_port",      "3306"},       // assuming default
+     // {"worker:worker-B.db_user",      "root"},       // assuming default
+     // {"worker:worker-B.db_password",  ""},           // assuming default
```

Comment:
should the commented out code get removed?

Reply:
The comments here are on purpose to illustrate what to expect in the unit test made downstream. I'm sorry, if these comments look confusing!